android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -691,7 +691,7 @@
 	{
 		"name": "Kremlin",
 		"cost": 500
-		"culture": 3,
+		,"culture": 3,
```

Comment:
:upside_down_face:

Reply:
I wish I could take credit for this chaos, but I think it was probably some kind of auto-format.